diff --git a/lib/generic-683db69b.d.ts b/lib/generic-683db69b.d.ts
index 31fbe16e2ebcc1d3af48cd6944d6416a67521f30..c17c6bad701a730cfcedd1bfa5367debf99c8d8e 100644
--- a/lib/generic-683db69b.d.ts
+++ b/lib/generic-683db69b.d.ts
@@ -114,8 +114,8 @@ interface GenericMessenger<TProtocolMap extends Record<string, any>, TMessageExt
      * @param data The data to send with the message.
      * @param args Different messengers will have additional arguments to configure how a message gets sent.
      */
-    sendMessage<TType extends keyof TProtocolMap>(type: TType, ...args: GetDataType<TProtocolMap[TType]> extends undefined ? [data?: undefined, ...args: TSendMessageArgs] : never): Promise<GetReturnType<TProtocolMap[TType]>>;
-    sendMessage<TType extends keyof TProtocolMap>(type: TType, data: GetDataType<TProtocolMap[TType]>, ...args: TSendMessageArgs): Promise<GetReturnType<TProtocolMap[TType]>>;
+    sendMessage<TType extends keyof TProtocolMap>(this: void, type: TType, ...args: GetDataType<TProtocolMap[TType]> extends undefined ? [data?: undefined, ...args: TSendMessageArgs] : never): Promise<GetReturnType<TProtocolMap[TType]>>;
+    sendMessage<TType extends keyof TProtocolMap>(this: void, type: TType, data: GetDataType<TProtocolMap[TType]>, ...args: TSendMessageArgs): Promise<GetReturnType<TProtocolMap[TType]>>;
     /**
      * Trigger a callback when a message of the requested type is received. You cannot setup multiple
      * listeners for the same message type in the same JS context.
@@ -125,11 +125,11 @@ interface GenericMessenger<TProtocolMap extends Record<string, any>, TMessageExt
      * @param type The message type to listen for. Call `sendMessage` with the same type to trigger this listener.
      * @param onReceived The callback executed when a message is received.
      */
-    onMessage<TType extends keyof TProtocolMap>(type: TType, onReceived: (message: Message<TProtocolMap, TType> & TMessageExtension) => void | MaybePromise<GetReturnType<TProtocolMap[TType]>>): RemoveListenerCallback;
+    onMessage<TType extends keyof TProtocolMap>(this: void, type: TType, onReceived: (message: Message<TProtocolMap, TType> & TMessageExtension) => void | MaybePromise<GetReturnType<TProtocolMap[TType]>>): RemoveListenerCallback;
     /**
      * Removes all listeners.
      */
-    removeAllListeners(): void;
+    removeAllListeners(this: void): void;
 }
 
 export { BaseMessagingConfig as B, GenericMessenger as G, Logger as L, MaybePromise as M, NamespaceMessagingConfig as N, ProtocolWithReturn as P, RemoveListenerCallback as R, GetDataType as a, GetReturnType as b, Message as c };
